src/Riok.Mapperly/Diagnostics/DiagnosticDescriptors.cs

```diff
@@ -269,4 +269,13 @@ internal static class DiagnosticDescriptors
         DiagnosticCategories.Mapper,
         DiagnosticSeverity.Info,
         true);
+
+    public static readonly DiagnosticDescriptor NotAllEnumValuesMapped = new DiagnosticDescriptor(
+        "RMG034",
+        "Not all enum values in the target enum are mapped",
+        "Not all enum values in the target enum are mapped from {0} to {1}",
```

Comment:
Instead of one generic diagnostic, I think `Enum member {0} on {1} not found on source enum {2}` and `Enum member {0} on {1} not found on target enum {2}` diagnostics help a lot during troubleshooting. It would also align with how properties diagnostics are reported.

Reply:
Agreed, fixed now.